src/components/API.js

```diff
@@ -25,7 +25,7 @@ export default class Api {
       },
     });
   }
-  updateProfileImage() {
```

Comment:
**Might be better**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`